test/lib/resources/kube.go

```diff
@@ -158,6 +159,35 @@ func EventTransformationPod(name string, event *cetest.CloudEvent) *corev1.Pod {
 	}
 }
 
+// EventTransformationPodV2 creates a Pod that transforms events received receiving as arg a cloudevents sdk2 Event
+// TODO(nlopezgi): remove EventTransformationPod above and rename this method to
+// EventTransformationPod once other tests that use sdk1 and depend on this method are migrated.
+func EventTransformationPodV2(name string, event *cloudevents.Event) *corev1.Pod {
```

Comment:
How about renaming the old method as `DeprecatedEventTransformationPod` and naming this one as it should be `EventTransformationPod`?

And i think here you don't need a pointer to event, but just `cloudevents.Event`

Reply:
done